requirements.txt

```diff
@@ -64,3 +64,4 @@ urllib3==1.26.5
 webencodings==0.5.1
 websocket-client==0.57.0
 zipp==3.3.1
+yaspin==2.0.0
```

Comment:
You'll also need to add this to setup.py so that it's installed when build-magic is installed.

Reply:
Done.